ParkPlanner/Park.cs

```diff
@@ -51,10 +51,10 @@ public partial class Datum
         public string Description { get; set; }
 
         [JsonProperty("latitude")]
-        public string Latitude { get; set; }
```

Comment:
4. Use double or decimal for Coordinates:
     Use double or decimal instead of string, if the properties Latitude and Longitude in the Datum class represent coordinates.

Reply:
Actually, I prefer Strings for lat/lng, because of Floating Point Arithmetic.